translate/storage/test_xliff.py

```diff
@@ -746,9 +747,8 @@ def test_addlocation():
 """
         xlifffile = xliff.xlifffile.parsestring(xlfsource)
         xlifffile.units[0].addlocation("some-directory/on-some-test/test.file2:333")
-
         locations = xlifffile.units[0].getlocations()
-
+        locations = ", ".join(x for x in locations if x is not None)
```

Comment:
Remove `None` filtering here:

```suggestion
        locations = ", ".join(locations)
```